src/scripts/Viewer.vue

```diff
@@ -137,6 +137,10 @@ export default {
 				return;
 			});
 		},
+
+		_encodeURI(uri) {
+			return encodeURI(uri).replace('#', encodeURIComponent('#'));
```

Comment:
did you compare with the files app in core and what logic it uses to avoid problems with hash signs in folders ?

I don't remember having such explicit repacement of hashes in URLs, so I suspect there is likely a better way (and maybe some utility function in core to encode?)

Reply:
Yes there is, it's calles `OC.encodePath()` but it only works with paths, wich leads to re-encoding if used prior to `encodeURI()`. So I avoided it by simply going for the "evil" char #

Reply:
I could work around it by dropping `encodeURI()` and going for `OC.encodePath()` only, but both behave differently.

Reply:
On second thought, I'll go for pure `OC.encodePath()`